shared/src/test/scala/org/adridadou/openlaw/vm/OpenlawExecutionEngineSpec.scala

```diff
@@ -2763,6 +2763,8 @@ class OpenlawExecutionEngineSpec extends FlatSpec with Matchers {
     text shouldBe s"""<p class="no-section"><br />hello world1,234 hello worldcfc2206eabfdc5f3d9e7fa54f855a8c15d196c05 hello world${nowFormatted}<br />1234hello world 0xcfc2206eabfdc5f3d9e7fa54f855a8c15d196c05hello world ${nowFormatted}</p>"""
   }
 
+  it should "not eat carriage returns" in {}
```

Comment:
Did you intend to write this test? @adridadou or perhaps it should be removed?

Reply:
you're right. removed!